cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,17 @@
+/// <reference types="cypress" />
+
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
 
+    cy.visit('user/register');
+    cy.get(':nth-child(1) > .form-control').type(user.username);
+    cy.get(':nth-child(2) > .form-control').type(user.email);
+    cy.get(':nth-child(3) > .form-control').type(user.password);
+    cy.get('.btn').click();
+    cy.get(':nth-child(4) > .nav-link');
+    cy.should('contain', user.username.toLowerCase());
```

Comment:
This assertion seems to be incomplete. `cy.should` is not a valid command. You probably intended to use it on a previously yielded subject from a command like `cy.get`. If you're trying to assert that the username exists in the DOM, the correct syntax would be `cy.get('yourSelector').should('contain', user.username.toLowerCase())`.